core/storage/collection/gae_models.py

```diff
@@ -144,7 +144,9 @@ def get_all_non_private_commits(
                 'max_age must be a datetime.timedelta instance or None.')
 
         query = cls.query(
-            cls.post_commit_is_private == False)  # pylint: disable=singleton-comparison
+            cls.post_commit_is_private # pylint: disable=singleton-comparison
```

Comment:
Ditto -- can't "== False" be tacked on to the end of this line? Or does Black force it onto a new line when you do that?